packages/streamline/src/components/cards/card-feedback/components/EmojiRating.tsx

```diff
@@ -0,0 +1,36 @@
+import { Pressable } from 'react-native-ama';
+
+import { Box } from '../../../../primitives/box/box';
+import { Icon } from '../../../../primitives/icon/icon';
+import { IconsName } from '../../../../types';
+
+export interface EmojiRatingProps {
+  currentRating: number;
```

Comment:
Shouldn't `currentRating` be handled at card-feedback level as a state and not as an external prop?

Reply:
Maybe we could still enable the outside world to manipulate it by exposing a ref if you think of use cases that we have currently

Reply:
Honestly, I don't see the benefit of having the state directly in the component. Because we will need to have a state on the luko project to store the rating. So we will create two state value for the currentRating

Reply:
example of a CardFeedback implementation in the project:

 ```
const LukoffFeedbackScreen: NavigationFunctionComponent<
  LukoffFeedbackScreenProps
> = ({ componentId, answers, contract }) => {
  const [rating, setRating] = useState<number>();
  ...

  return (
    ...
          {isBadExperience ? (
            <Text variant="body" color="GREY_700">
              {i18n.t('cover.resume.terminate.feedback.body')}
            </Text>
          ) : (
            <Box>
              <LukoUIThemeProvider>
                <FeedbackSmileRating
                  selected={selectedSmileLevel}
                  onSelect={setRating}
                  androidLabel={i18n.t(
                    'cover.resume.terminate.feedback.lukoExperience'
                  )}
                />
              </LukoUIThemeProvider>
            </Box>
          )}
       ...
  );
};

export default LukoffFeedbackScreen;
``